testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -126,16 +127,22 @@ impl BitcoinRegtestController {
         }        
     }
 
-    fn setup_indexer_runtime(&mut self) -> (Burnchain, BitcoinIndexer) {
+    fn setup_burnchain(&self) -> Burnchain {
         let network = "regtest".to_string();
```

Comment:
After rebasing with next, can you make sure that the hard-coded "regtest" is replaced by `config.burnchain.get_bitcoin_network()`? It's important for our ongoing preparation with Xenon.

Reply:
Thanks for the heads-up. Merged `next` and patched this in 4a6a38a30